documentation/modules/ref-operator-topic.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-topic-operator.adoc
+
+[id='ref-operator-topic-{context}']
+= Kafka topic resource
+
+The `KafkaTopic` resource is used to configure topics, including the number of partitions and replicas.
+
+The full schema for `KafkaTopic` is described in xref:type-KafkaTopic-reference[`KafkaTopic` schema reference].
+
+= Identifying a Kafka cluster for topic handling
+
+A `KafkaTopic` resource includes a label that defines the appropriate name of the Kafka cluster (derived from the name of the `Kafka` resource) to which it belongs.
+
+For example:
+
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: KafkaTopic
+metadata:
+  name: topic-name-1
+  labels:
+    strimzi.io/cluster: my-cluster
+----
+
+The label is used by the Topic Operator to identify the `KafkaTopic` resource and create a new topic, and also in subsequent handling of the topic.
+
+If the label does not match the Kafka cluster, the Topic Operator cannot identify the `KafkaTopic` and the topic is not created.
+
+= Handling changes to topics
```

Comment:
Same as above, should this be `==` ... this seems to be on a lot of places ... not sure if this is right or wrong.

Reply:
Yeah. I've moved this down a level. I originally left it like that to present these sections in the TOC, but I don't think its necessary.